lib/components/settings/nextcloud_profile.dart

```diff
@@ -116,10 +116,25 @@ class _NextcloudProfileState extends State<NextcloudProfile> {
                   ),
                   tooltip: t.settings.resyncEverything,
                   onPressed: () async {
-                    final allFiles = await FileManager.getAllFiles();
+                    final allFiles = await FileManager.getAllFiles(false);
                     Prefs.fileSyncResyncEverythingDate.value = DateTime.now();
                     for (final file in allFiles) {
                       FileSyncer.addToUploadQueue(file);
+                      // upload also all file assets if any
```

Comment:
Rather than using this for-loop which is quite slow, I think we should add a `{bool includeAssetFiles = false}` parameter into `FileManager.getChildrenOfDirectory`

Reply:
Yes, it seems to be good idea.